docs/UI/SetMediaClockTimer/index.md

```diff
@@ -22,12 +22,15 @@ The UI.SetMediaClock timer request indicates either an initial value for the med
         * Start counting up or down from the requested `startTime` value at the specified `countRate` with a step of 1 second;
         * Continue counting up or down until:
             * The next request of _SetMediaClockTimer_ with appropriate parameters comes;
+            * `endTime` is reached, if provided.
             * Zero is reached in the case of COUNTDOWN.   
     * PAUSE mode:   
         * Pause the timer that is counting up or down;   
         * If `startTime` or `endTime` parameters are provided, the values must be updated on the HMI.   
+    * RESUME mode:
+        * Resume the timer that was previously paused at the specified `countRate`;
     * CLEAR mode:    
-        * Clear `startTime` to 00:00:00 in the case that the `startTime` parameter is not provided in the request, otherwise, `startTime` must be updated with a new value. It is up to HMI to determine the way the media clock timer is cleared: either to remove it from display or to set it to zero.   
+        * Clear `startTime` to 00:00:00 in the case that the `startTime` parameter is not provided in the request, otherwise, `startTime` must be updated with a new value. It is up to HMI to determine the way the media clock timer is cleared: either to remove it from display or to set it to zero.
```

Comment:
```suggestion
        * Clear `startTime` to 00:00:00 in the case that the `startTime` parameter is not provided in the request, otherwise, `startTime` must be updated with a new value. It is up to the HMI to determine the way the media clock timer is cleared: either to remove it from display or to set it to zero.
```